app/views/flood_risk_engine/enrollments/steps/_local_authority_address.html.erb

```diff
@@ -1 +1,34 @@
-local_authority_address
+<%= t('.postcode_hint') %>
+<p>
+  <%= form.post_code %>
+  <%= link_to t('.change'),
+              enrollment_step_path(form.enrollment, form.enrollment.previous_step),
+              class: 'change-postcode-button'
+  %>
+</p>
+
+<%= f.hidden_field :post_code, value: form.post_code %>
+
+<label class="form-label" for="address_match_selection">
+  <%= t('.select_from_list_label') %>
+</label>
+
+<% form.address_list.unshift({ moniker: t('.num_addresses_found', num:  form.address_list.count) }) %>
+
+<fieldset>
+  <%= form_group_and_validation(form, :uprn) do %>
+    <%= f.select :uprn,  options_for_select( form.address_list.collect { |a| [ a[:moniker], a[:uprn] ] }), {},
+                 { include_blank: false,
```

Comment:
Instead of `include_blank: false` use:
```
prompt: t('.num_addresses_found', num:  form.address_list.count)
```
Then you don't have to add the list count to the list of addresses, and you can let the select form helper handle what to set the value as.